src/components/cards/cartCard/CartCard.js

```diff
@@ -17,12 +21,25 @@ export const CartCard = ({ productDetails, horz, newBadge }) => {
     qty,
   } = productDetails
 
-  const { cartDispatch } = useCart()
-
   useEffect(() => {
     qty >= 5 && Toast(`Only 5 unit(s) allowed in each order`, 'info')
   }, [qty])
 
+  const { cartDispatch } = useCart()
+
+  const { wishlistState, wishlistDispatch } = useWishlist()
+  const { wishlistItems } = wishlistState
+
+  const moveToWishlistHandler = () => {
+    if (wishlistItems.find((item) => item._id === _id)) {

```

Comment:
I think here you should be searching for the index in the wishlist array and not the value or item itself. So consider using .findIndex method and not .find

Reply:
Okay will try this.